使环境变量TZ在Docker中有效，并默认使用UTC时区

KUMAKAIHA opened the pull request:
I modified the Dockerfile to include setting the TZ variable, with the default UTC timezone. I also edited telemetry_db.php to add a timestamp for timezone configuration. Additionally, PHP now defaults to using UTC as the timezone when no TZ variable is passed.
I compiled a test Docker on my Mac, and everything runs fine on my Mac. However, the version compiled on Mac is for arm64, but I believe it shouldn't have a big impact on the amd64 version. I only tried adding the timezone as Shanghai and got the correct timezone on the query page. I hope to get more tests to ensure the stability of the project.

sstidl reviewed (changes requested):
pls remove .DS_Store

the timestamp col in sql is defined to insert current date-time. so it is not needed to set the timestamp in the insert statement.

KUMAKAIHA commented:
> 请删除。DS_商店
> 
> sql中的时间戳col被定义为插入当前日期和时间。因此不需要在插入语句中设置时间戳。

I am very sorry for uploading .DS_Store. Firstly, it was because I didn't know this hidden file is invisible in VSCode. Secondly, I was new to macOS at that time and didn't realize the system would create these files in every folder. So I have now deleted this file and updated the .gitignore file to prevent macOS users from continuing to upload .DS_Store. Regarding the date-time issue in SQL that you mentioned, I am not sure if this will affect MySQL, MSSQL, etc. However, the Docker version should be using SQLite by default. I tried modifying the system time inside the container and other methods, but I couldn't correct the database time. Therefore, I took a different approach by having PHP directly provide the correct timestamp to the database. As for normal deployment scenarios, I may need to investigate further and perhaps use other methods to correct the SQLite time. But at least in the current state, it supports Docker configuration of `tz` to use the correct local time, and I have already successfully tested it on a Debian server.
